src/conda.exe.spec

```diff
@@ -62,6 +62,13 @@ elif sys.platform == "darwin":
     ]
     extra_exe_kwargs["entitlements_file"] = os.path.join(HERE, "entitlements.plist")
 
+# Add .condarc file to bundle to configure channels
+# during the package building stage
+if "RECIPE_DIR" in os.environ:
```

Comment:
Should we check for `CONDA_BUILD` too?

Reply:
I don't see the benefit here. I would rather point this to the `.condarc` file so that we are not strongly coupled to conda build. We could, e.g., also use `pyinstaller` directly to build it.

What I could do is rename the variable to be more general (`CONDARC_LOCATION`) and set that environment variable in the `meta.yaml` file. That actually sounds better to me.